pkgs/development/python-modules/icoextract/default.nix

```diff
@@ -22,10 +22,15 @@ buildPythonPackage rec {
     "icoextract"
   ];
 
+  postInstall = ''
+    mkdir -p $out/share/thumbnailers
+    substituteAll ${./exe-thumbnailer.thumbnailer} $out/share/thumbnailers/exe-thumbnailer.thumbnailer
```

Comment:
There is a reason to not copy from source https://github.com/jlu5/icoextract/blob/master/exe-thumbnailer.thumbnailer ?

Reply:
The thumbnailer isn't included in the pypi archive https://pypi.org/project/icoextract/#files

Reply:
Oh gotcha, okay, LGTM.